.golangci.yml

```diff
@@ -9,6 +9,8 @@ run:
     - third_party$
     - vendor$
   skip-dirs-use-default: false
+  build-tags:
+  - all
```

Comment:
What's this for?

Reply:
A load of the test files have build tags on them so they only build (or lint) if that tag is set. e.g. `tests/integration/aliases/aliases_go_test.go` has `//go:build (go || all) && !xplatform_acceptance` at the top of it.

`all` is just the tag we use to save us having to write `python,nodejs,go` longhand to compile these files.